scripts/deploy/nlp/deploy_triton.py

```diff
@@ -375,14 +222,10 @@ def get_nemo_deployable(args):
     return MegatronLLMDeployable(args.nemo_checkpoint, args.num_gpus)
 
 
-def nemo_deploy(argv):
-    args = get_args(argv)
-
-    if args.debug_mode:
-        loglevel = logging.DEBUG
-    else:
-        loglevel = logging.INFO
+def nemo_deploy():
+    args = get_args(sys.argv[1:])
```

Comment:
my point was that you can completely remove passing `argv` around up until running `args = parser.parse_args()` w/o any param list as by default parser will use `sys.argv[1:]` (in both scripts `deploy_triton.py` and `export_to_trt_llm.py`)